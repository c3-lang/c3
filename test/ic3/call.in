Module.ident(arg, arg, arg)
a(b)
