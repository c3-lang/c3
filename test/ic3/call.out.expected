quote test()

quote test(1)

quote test(1, 2)

quote test(1, 2, 3)

quote Test.test()

quote Test.test(1)

quote Test.test(1, 2)

quote Test.test(1, 2, 3)

